src/contexts.js

```diff
@@ -1,4 +1,8 @@
-import { createContext } from 'react';
+// @flow
 
-export const PriorityContext = createContext(null);
-export const TimeRangeContext = createContext([0, 0]);
+// import { createContext } from 'react';
+
+// TODO: Figure out what these contexts are for. They were added in the
+// prototype but were unused.
+// export const PriorityContext = createContext(null);
+// export const TimeRangeContext = createContext([0, 0]);
```

Comment:
There's more unused code I've encountered in the codebase so let's comment all of it or none of it?
We can keep the `TODO` though to keep this in mind for later.

Reply:
Oh yeah, this is just a first batch, I'm sure there's a lot of dead code that I missed. We can comment all of it in future PRs